nodebuilder/share/cmd/share.go

```diff
@@ -158,8 +174,8 @@ var getShare = &cobra.Command{
 }
 
 var getEDS = &cobra.Command{
-	Use:   "get-eds [extended header]",
-	Short: "Gets the full EDS identified by the given extended header",
+	Use:   "get-eds [block height or hash]",
```

Comment:
```suggestion
	Use:   "get-eds (height | hash)",
```

The usage of square brackets was wrong. It should be used to show optionality while here the arguments is mandatory. Also, for mutually exclusive arguments | must be used.

Reply:
See https://pubs.opengroup.org/onlinepubs/000095399/basedefs/xbd_chap12.html#tag_12_01